### DISCUSSION TITLE
OnTitle() C# API cannot successfully excute with the ItemsAdder plugin`s title

### DISCUSSION THREAD

**Discussion by Sagittaro** [ORelio/Minecraft-Console-Client#2698] [Q&A]
I tried to write a special autofish bot for my server.This server use new fishing plugin to replace the Vanilla.While baiting the fish, it will send a title depends on the ItemAdders plugin.The titles are as follow:
![2024-03-05_22 31 43](https://github.com/MCCTeam/Minecraft-Console-Client/assets/51532982/a4e99be1-5ed0-4fe1-ab3d-52047c30dd4b)
![2024-03-05_22 32 59](https://github.com/MCCTeam/Minecraft-Console-Client/assets/51532982/d8dcbbc5-8721-45aa-95c3-58947eedd880)
I found the plugin's source, and read the gitbook, it says fishing title follow the config format as follow:
first title's format:
![1](https://github.com/MCCTeam/Minecraft-Console-Client/assets/51532982/e7374373-8524-4ce0-9953-0cad7adf736f)
`https://mo-mi.gitbook.io/xiaomomi-plugins/plugin-wiki/customfishing/minigame/accruate-click`
second title's format:
![2](https://github.com/MCCTeam/Minecraft-Console-Client/assets/51532982/26137f73-86f5-437d-83c9-109065481791)
` https://mo-mi.gitbook.io/xiaomomi-plugins/plugin-wiki/customfishing/minigame/tension`

I wrote a little script try to get the fish title data sended from server and displayed in my console, but received nothing.
![js](https://github.com/MCCTeam/Minecraft-Console-Client/assets/51532982/90a3f5f7-ec0e-47f8-a0b5-b56ae2c78e8c)

![QQ图片20240305233923](https://github.com/MCCTeam/Minecraft-Console-Client/assets/51532982/1cff49b5-a77b-4807-91fa-a03dad3c8cc9)
How can I get the title data from the server?

**Comment by Nekiplay** [comment]
/// <summary>
        /// Called when received a title from the server
        /// <param name="action"> 0 = set title, 1 = set subtitle, 3 = set action bar, 4 = set times and display, 4 = hide, 5 = reset</param>
        /// <param name="titletext"> title text</param>
        /// <param name="subtitletext"> suntitle text</param>
        /// <param name="actionbartext"> action bar text</param>
        /// <param name="fadein"> Fade In</param>
        /// <param name="stay"> Stay</param>
        /// <param name="fadeout"> Fade Out</param>
        /// <param name="json"> json text</param>
        public virtual void OnTitle(int action, string titletext, string subtitletext, string actionbartext, int fadein, int stay, int fadeout, string json) { }
Example:
![image](https://github.com/MCCTeam/Minecraft-Console-Client/assets/35975332/1328d20f-f34b-43df-8612-3a6f6d6e6fc5)

**Reply by Sagittaro** [reply]
Thanks for your help. I tried your suggestions, but MCC console still received nothing but only MCC AutoFish script`s log :(
